src/Components/SubMenu.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import '../Styling/SubMenu.css';
+
+
+const SubMenu = ({ item }) => {
+    const [subnav, setSubnav] = useState(false);
+  
+    const showSubnav = () => setSubnav(!subnav);
+  
+    return (
+      <>       
+            <Link className='Menu_item_links' 
+            to={item.path} onClick={item.subNav && showSubnav}>
+            <div className='Menu_icons'>
+                <div className='Menu_items'>{item.title}</div>
```

Comment:
Title was used as value for each page name to be displayed. I  went ahead and changed 'title' to 'page' so it could be more understandable. It displays each page on the navbar